organisms/organisation-unit-tree/search.feature

```diff
@@ -15,6 +16,7 @@ Feature: Search the organisation unit matching the search term
     Given the user searches the organisation unit tree
     When the result contains a unit in the sub tree
     Then then all units in the path will be shown
+    And the units in the path that do not match the search term will be visually different from matched units
```

Comment:
I think this is too design specific, it could be "disabled", but visually different is the job of the design, not the behavior (although is seems to be very vague in this case, def. debatable)

Reply:
True. They shouldn't be disabled (use case: search for the child in order to select it's parent). But yes, this is too design specific. Removed.